src/platform/unittest/DataPathTest.cpp

```diff
@@ -18,6 +18,7 @@
 #endif
 
 extern bool UseDuoNic;
+#define QUIC_EXECUTION_CONFIG_FLAG_XDP (QUIC_EXECUTION_CONFIG_FLAGS)0x0004
```

Comment:
Instead of this, simply define `QUIC_API_ENABLE_PREVIEW_FEATURES` above, before all the includes.